Resources/Prototypes/Entities/Tiles/bananium.yml

```diff
@@ -0,0 +1,73 @@
+- type: entity
+  id: FloorBananiumEntity
+  name: bananium floor
+  placement:
+    mode: SnapgridCenter
+  components:
+  - type: Clickable
+  - type: Sprite
+    netsync: false
+    sprite: Tiles/Misc/bananium.rsi
+    drawdepth: FloorTiles
+    layers:
+      - state: bananium
+  - type: Icon
+    sprite: Tiles/Misc/bananium.rsi
+    state: bananium
```

Comment:
Don't think icon is needed.

Reply:
isnt it needed for crafting? will test it out

Reply:
as above